components/SiteHeader/CategoriesList.js

```diff
@@ -1,39 +1,18 @@
 import Link from "next/link";
-
-// temp thing - delete later
-const categories = [
-  {
-    id: 1,
-    title: "Mental Health",
-    description: "Learn about tips, mental health issues",
-    link: "/categories/mental-health",
-  },
-  {
-    id: 2,
-    title: "Life Issues",
-    description: "Learn about life issues, product updates and company culture",
-    link: "/categories/life-issues",
-  },
-  {
-    id: 3,
-    title: "Social & Relationships",
-    description:
-      "Learn about social and relationships issues, product updates and company culture",
-    link: "/categories/social-relationship",
-  },
-];
+import useSWR from "swr";
 
 const CategoriesList = () => {
+  const { data: categories } = useSWR("/api/categories");
   return (
     <div>
-      {categories.map((category) => (
-        <Link href={category.link} key={category.id}>
-          <a className="block hover:bg-gray-200 p-6">
-            <h3 className="font-extrabold">{category.title}</h3>
-            <p className="text-gray-500">{category.description}</p>
-          </a>
-        </Link>
-      ))}
+      {categories &&
+        categories.data.map(({ id, slug, name }) => (
```

Comment:
This looks good to me and I'm sure you'll know the answer to this better than I would, but is it definitely `categories.data.map` and not `categories.map`?

I can't know which one it is, because it depends on what you're getting back from your requests and I don't know how these objects are structured, but I have a feeling I've seen both used in the PRs I've looked at.

If it works, you're doing the right thing.

Reply:
Hi @ARTurnerGit Thank you for the feedback. Yes, It definitely is categories. data as I am getting the right data from it :)